### PR TITLE
Feat/svm

### PR DESCRIPTION
**Pull request by AnibalITQ**
En los logs vienen los resultados históricos de las 3 modalidades que intentamos

### CODE REVIEW

**Comment by AnibalITQ** [comment]
aquí un pequeño resumen de los resultados

2025-03-04 00:52:15,632 - svm_monitor - INFO - Caracter�sticas guardadas en train_dataset_simple.csv
2025-03-04 00:52:16,265 - svm_monitor - INFO - Caracter�sticas guardadas en test_dataset_simple.csv
2025-03-04 00:52:16,265 - svm_monitor - INFO - Training SVM base model...
2025-03-05 15:49:27,003 - svm_monitor - INFO - Training SVM base model...
2025-03-05 16:13:32,806 - svm_monitor - INFO - Training SVM base model...
2025-03-06 01:05:05,361 - svm_monitor - INFO - Model: SVM
2025-03-06 01:05:05,361 - svm_monitor - INFO - Training time: 17205.3121 seconds
2025-03-06 01:05:05,361 - svm_monitor - INFO - Prediction time: 210.9861 seconds
2025-03-06 01:05:05,361 - svm_monitor - INFO - Accuracy: 0.7943
2025-03-06 01:05:05,361 - svm_monitor - INFO - Weighted precision: 0.7943
2025-03-06 01:05:05,361 - svm_monitor - INFO - Weighted recall: 0.7943
2025-03-06 01:05:05,361 - svm_monitor - INFO - Weighted F1-score: 0.7943
2025-03-06 01:05:05,377 - svm_monitor - INFO - Informe de clasificaci�n:
              precision    recall  f1-score   support

           0       0.79      0.80      0.80     16391
           1       0.80      0.79      0.79     16377

    accuracy                           0.79     32768
   macro avg       0.79      0.79      0.79     32768
weighted avg       0.79      0.79      0.79     32768

2025-03-06 01:05:05,377 - svm_monitor - INFO - Matriz de confusi�n:
[[13097  3294]
 [ 3446 12931]]
2025-03-06 17:49:53,042 - svm_monitor - INFO - Model: SVM-linear
2025-03-06 17:49:53,042 - svm_monitor - INFO - Training time: 60175.0176 seconds
2025-03-06 17:49:53,048 - svm_monitor - INFO - Prediction time: 112.6463 seconds
2025-03-06 17:49:53,048 - svm_monitor - INFO - Accuracy: 0.7784
2025-03-06 17:49:53,048 - svm_monitor - INFO - Weighted precision: 0.7805
2025-03-06 17:49:53,048 - svm_monitor - INFO - Weighted recall: 0.7784
2025-03-06 17:49:53,048 - svm_monitor - INFO - Weighted F1-score: 0.7780
2025-03-06 17:49:53,057 - svm_monitor - INFO - Informe de clasificaci�n:
              precision    recall  f1-score   support

           0       0.80      0.74      0.77     16391
           1       0.76      0.82      0.79     16377

    accuracy                           0.78     32768
   macro avg       0.78      0.78      0.78     32768
weighted avg       0.78      0.78      0.78     32768

2025-03-06 17:49:53,060 - svm_monitor - INFO - Matriz de confusi�n:
[[12058  4333]
 [ 2928 13449]]
